tests/integration/test_vid_club.py

```diff
@@ -49,12 +52,46 @@ def test_build_query_wrong_source():
         )
 
 
+@pytest.mark.intervals
+def test_intervals_split():
+    from_date = "2022-01-01"
+    to_date = "2022-01-19"
+    days_interval = 5
+    expected_starts = ["2022-01-01", "2022-01-06", "2022-01-11", "2022-01-16"]
+    expected_ends = ["2022-01-06", "2022-01-11", "2022-01-16", "2022-01-19"]
+    starts, ends = vc.intervals(
+        from_date=from_date, to_date=to_date, days_interval=days_interval
+    )
+
+    assert starts == expected_starts
+    assert ends == expected_ends
+
+
+@mock.patch("viadot.sources.vid_club.VidClub.check_connection")
+@pytest.mark.connection_check
+def test_check_connection(mock_function):
+    mock_first_output = "Mocked Output"
+    mock_second_output = "Mocked Output"
+    mock_function.return_value = mock_first_output, mock_function.return_value[1]
+
+    response, first_url = vc.check_connection(
+        source="jobs",
+        from_date="2023-03-24",
+        to_date="2023-03-24",
+        url="test",
+        items_per_page=1,
+    )
+    print(response)
+    print(first_url)
+    assert isinstance(first_url, str)
+    assert len(first_url) > len(mock_second_output)
+
+
```

Comment:
Please add docstring to test function that will shortly summarize what they are testing